qemu/tests/block_copy.py

```diff
@@ -265,6 +265,10 @@ def get_image_file(self):
             if isinstance(blocks, str):
                 image_regex = '%s.*\s+file=(\S*)' % self.device
                 image_file = re.findall(image_regex, blocks)
+                if image_file:
+                    return image_file[0]
+                image_file = re.findall("%s[^:]+: (\S*)" % self.device,
```

Comment:
Based on the commit message, I'm assuming that QEMU does support spaces in filenames.  What prevents us from *also* support it?

Reply:
time :-) Anyway I'll improve it in v2:

```
(qemu) info block
ide0-hd0 (#block184): a b (raw)
    Attached to:      /machine/unattached/device[18]
    Cache mode:       writeback
```